craft_providers/multipass/multipass_provider.py

```diff
@@ -100,8 +100,9 @@ def name(self) -> str:
     ubuntu.BuilddBaseAlias.KINETIC: RemoteImage(
         remote=Remote.RELEASE, image_name="kinetic"
     ),
-    ubuntu.BuilddBaseAlias.LUNAR: RemoteImage(
-        remote=Remote.RELEASE, image_name="lunar"
+    ubuntu.BuilddBaseAlias.LUNAR: RemoteImage(remote=Remote.DAILY, image_name="lunar"),
```

Comment:
Why did you change lunar from `release` to `daily`?

Reply:
Does it have the same issue as buildd image or not?

Reply:
It shouldn't.  

The Multipass `daily` and `release` remotes are for "regular" cloud images and the `snapcraft` remote are for buildd images.

Reply:
So we should use `release` for now and move to `snapcraft` later?

Reply:
Correct

Reply:
It has been updated.

Reply:
I don't think we are going to use mantic for a while.